apps/api/routes.py

```diff
@@ -234,10 +234,10 @@ def delete_group(group_id):
     if not group or group.is_deleted:
         return {"status": "fail", "result": "Group not found"}, 404
 
-    if any([
-        current_user.category not in ["admin", "teacher"],
```

Comment:
this validation was duplicated with the line 230 above. Took the opportunity to fix this.